clients/lnurl.js

```diff
@@ -1,12 +1,31 @@
 
-const { requestInvoice } = require('lnurl-pay')
+const { requestInvoice, utils } = require('lnurl-pay')
+const { isOnionUrl, decodeUrlOrAddress } = utils
+const tor_axios = require('tor-axios');
+const config = require('../config.json')
+
+const tor = tor_axios.torSetup({
+    ip: config.TOR_HOST,
+    port: config.TOR_PORT || 9050,
+})
+
+async function torGet({ url, params }) {
+    console.log({ url, params })
+    const resp = await tor.get(url, { params })
+    return resp.data
+}
 
 async function fetchInvoice({ lnUrlOrAddress, paymentAmountSats }) {
+    const url = decodeUrlOrAddress(source.lnurl_or_lnaddress)
```

Comment:
Should this read instead?
```
    const url = decodeUrlOrAddress(lnUrlOrAddress)
```
I think `source` is what is throwing my error...

Reply:
yup good catch

Reply:
the dangers of not having any tests!